app/src/main/java/com/android/streetworkapp/ui/event/AddEvent.kt

```diff
@@ -115,22 +119,31 @@ fun AddEventScreen(
           }
       FloatingActionButton(
           onClick = {
-            if (event.title.isEmpty()) {
+            if (event.date.toDate() < Calendar.getInstance().time) {
+              Toast.makeText(
+                      context, "Please select a valid time in the future", Toast.LENGTH_SHORT)
+                  .show()
+            } else if (event.title.isEmpty()) {
               Toast.makeText(context, "Please fill the title of the event", Toast.LENGTH_SHORT)
                   .show()
             } else {
               eventViewModel.addEvent(event)
+              parkViewModel.currentPark.value?.let {
+                parkViewModel.addEventToPark(it.pid, event.eid)
```

Comment:
You are right, but this value might be null. But you are right, I will update this line and use the event.parkId, since in the worst case it has the default parkId value of a park. Thank you for your suggestion!